porcupine/plugins/pastebin.py

```diff
@@ -246,7 +247,66 @@ def pasting_done_callback(
         )
 
 
+def ask_are_you_sure(filename: str | None, paste_class: type[Paste]) -> bool:
+    window = tkinter.Toplevel()
```

Comment:
```suggestion
    window = tkinter.Toplevel()
    window.transient(get_main_window())
```
Just so it doesn't appear as a separate icon on my taskbar ;)